lib/skate_web/templates/page/index.html.eex

```diff
@@ -1 +1 @@
-<div id="app" data-user-token="<%= assigns[:user_token] %>"></div>
+<div id="app" data-user-token="<%= assigns[:user_token] %>" data-laboratory-features="<%= Jason.encode!(assigns[:laboratory_features]) %>"></div>
```

Comment:
`:laboratory_features` will always be assigned, right? in which case it's safe to use `@laboratory_features` in this template